Havit.Blazor.Components.Web.Bootstrap/wwwroot/defaults.css

```diff
@@ -152,7 +152,7 @@
 	--hx-multi-select-dropdown-menu-width: 							100%;
 
 	/* TagInput */
-	--hx-input-tags-tag-margin: 									0 .25rem 0 0;
+	--hx-input-tags-tag-margin: 									.25rem .25rem .25rem 0;
```

Comment:
Agree with the intension but lets do it with a gap on hx-input-tags-control class. The css var name will be --hx-input-tags-tag-gap.

Reply:
... please update the documentation as well.